docs/_documentation/02_features/built-in-functions.md

```diff
@@ -0,0 +1,437 @@
+---
+layout: collection-browser-doc
+title: Built-in functions
+category: Features
+categories_url: features
+excerpt: Terragrunt allows you to use built-in functions anywhere in `terragrunt.hcl`, just like Terraform.
+tags: ["functions"]
+order: 207
+nav_title: Documentation
+nav_title_link: /documentation/
+---
+## Built-in Functions
+
+Terragrunt allows you to use built-in functions anywhere in `terragrunt.hcl`, just like Terraform\! The functions currently available are:
+
+  - [All Terraform built-in functions](#terraform-built-in-functions)
+
+  - [find\_in\_parent\_folders()](#find_in_parent_folders)
+
+  - [path\_relative\_to\_include()](#path_relative_to_include)
+
+  - [path\_relative\_from\_include()](#path_relative_from_include)
+
+  - [get\_env(NAME, DEFAULT)](#get_env)
+
+  - [get\_terragrunt\_dir()](#get_terragrunt_dir)
+
+  - [get\_parent\_terragrunt\_dir()](#get_parent_terragrunt_dir)
+
+  - [get\_terraform\_commands\_that\_need\_vars()](#get_terraform_commands_that_need_vars)
+
+  - [get\_terraform\_commands\_that\_need\_input()](#get_terraform_commands_that_need_input)
+
+  - [get\_terraform\_commands\_that\_need\_locking()](#get_terraform_commands_that_need_locking)
+
+  - [get\_terraform\_commands\_that\_need\_parallelism()](#get_terraform_commands_that_need_parallelism)
+
+  - [get\_aws\_account\_id()](#get_aws_account_id)
+
+  - [get\_aws\_caller\_identity\_arn()](#get_aws_caller_identity_arn)
+
+  - [get\_aws\_caller\_identity\_user\_id()](#get_aws_caller_identity_user_id)
```

Comment:
Sanity check: are the `\` there to escape the `_`, so Markdown doesn't interpret that as italics?

Reply:
Correct, `_` makes it italics.